commit 50dc95d561a2552b0d76a9f91b38005195bf2974
Author: Philipp Zabel <philipp.zabel@gmail.com>
Date:   Sun Jan 23 11:16:53 2022 +0100

    drm/edid: remove non_desktop quirk for HPN-3515 and LEN-B800.
    
    Now that there is support for the Microsoft VSDB for HMDs, remove the
    non-desktop quirk for two devices that are verified to contain it in
    their EDID: HPN-3515 and LEN-B800.
    Presumably most of the other Windows Mixed Reality headsets contain it
    as well, but there are ACR-7FCE and SEC-5194 devices without it.
    
    Tested with LEN-B800.
    
    Signed-off-by: Philipp Zabel <philipp.zabel@gmail.com>
    Reviewed-by: Jani Nikula <jani.nikula@intel.com>
    Signed-off-by: Jani Nikula <jani.nikula@intel.com>
    Link: https://patchwork.freedesktop.org/patch/msgid/20220123101653.147333-2-philipp.zabel@gmail.com

diff --git a/drivers/gpu/drm/drm_edid.c b/drivers/gpu/drm/drm_edid.c
index 271b5616cfaf..c23ad8f0b3e9 100644
--- a/drivers/gpu/drm/drm_edid.c
+++ b/drivers/gpu/drm/drm_edid.c
@@ -214,9 +214,7 @@ static const struct edid_quirk {
 
 	/* Windows Mixed Reality Headsets */
 	EDID_QUIRK('A', 'C', 'R', 0x7fce, EDID_QUIRK_NON_DESKTOP),
-	EDID_QUIRK('H', 'P', 'N', 0x3515, EDID_QUIRK_NON_DESKTOP),
 	EDID_QUIRK('L', 'E', 'N', 0x0408, EDID_QUIRK_NON_DESKTOP),
-	EDID_QUIRK('L', 'E', 'N', 0xb800, EDID_QUIRK_NON_DESKTOP),
 	EDID_QUIRK('F', 'U', 'J', 0x1970, EDID_QUIRK_NON_DESKTOP),
 	EDID_QUIRK('D', 'E', 'L', 0x7fce, EDID_QUIRK_NON_DESKTOP),
 	EDID_QUIRK('S', 'E', 'C', 0x144a, EDID_QUIRK_NON_DESKTOP),